src/02-common-principles.md

```diff
@@ -152,6 +152,20 @@ correspond to a unique identifier of that subject, such as `01`.
 The same holds for the `session` entity with its `ses-` key and its `<label>`
 value.
 
+Session labels MUST consist only of alphanumeric characters `[a-zA-Z0-9]`
+and SHOULD be consistent across subjects.
+If numbers are used in session labels we RECOMMEND using zero padding
+(for example `ses-01`, `ses-11` instead of `ses-1`, `ses-11`). This makes
+results of alphabetical sorting more intuitive. Acquisition time of session can
+be defined in the [sessions file](03-modality-agnostic-files.md#sessions-file).
```

Comment:
I think can be significantly pruned, because the essential information about `<label>` is already given in common principles:

```
1.  **`<label>`** - an alphanumeric value, possibly prefixed with arbitrary
    number of 0s for consistent indentation, for example, it is `rest` in `task-rest`
    following `task-<label>` specification.
```

And together with this pruning, I think it will not be as bad of a break anymore as some may perceive it to be right now in this place.

Reply:
Feel free to remind me daily that this is supposed to be a specification not a page turner that reads like a best-seller. :wink:

Reply:
just a note: I think you have a valid point and that it's important to have a good reading flow - even for a specification. Perhaps my bar of what I consider good enough is lower :confused: That's why it'd be good to have at least one other person chime in :-)